conf/sidecar.yaml

```
@@ -193,6 +197,17 @@ access_control:
         #
         # other options are, io.vertx.ext.auth.mtls.impl.SpiffeIdentityExtractor.
         certificate_identity_extractor: org.apache.cassandra.sidecar.acl.authentication.CassandraIdentityExtractor
+      # JwtAuthenticationHandlerFactory adds support to authenticate users with their JWT tokens. It also includes
+      # supports for OpenID discovery.
+    - class_name: org.apache.cassandra.sidecar.acl.authentication.JwtAuthenticationHandlerFactory
+      parameters:
+        # Site for sidecar to dynamically retrieve the configuration information of an OpenID provider, without
+        # having to manually configure settings like issuer etc.
+        site: https://authorization.com
+        # Client Id is a unique identifier assigned by OpenID provider. It is used to identity applications/users
+        # trying to connect.
+        client_id: recognized_client_id
+        config_discover_interval: 1m
```

I think this is running too frequently. I suggest a default of 1 hour instead. What do you think?
```suggestion
        config_discover_interval: 1h
```